build/gen-qt-projectfile.bat

```diff
@@ -1,25 +1,25 @@
-@echo off
-
-set OLD_DIR=%CD%
-
-echo TRANSLATIONS = \
-for /r %1/share/locale/ %%a in (mscore_*.ts) do echo     %%a \
-echo.
-
-cd /d %1
-
-echo FORMS = \
-for /r %1 %%a in (*.ui) do echo     %%a \
-echo.
-
-echo SOURCES = \
-for /r %1 %%a in (*.cpp) do echo     %%a \
-echo.
-echo.
-
-echo RESOURCES = \
-echo     ./mu4/scenes/palette/palette.qrc \
-echo     ./telemetry/telemetry_resources.qrc \
-echo.
-
-cd /d %OLD_DIR%
+@echo off
```

Comment:
Is that (I assume it to be cr vs. cr/lf) the reason why `lupdate` doesn't really work on Windows?